bindings/wallet-c/src/lib.rs

```diff
@@ -754,3 +822,15 @@ pub extern "C" fn iohk_jormungandr_wallet_delete_conversion(conversion: Conversi
 pub extern "C" fn iohk_jormungandr_wallet_delete_proposal(proposal: ProposalPtr) {
     wallet_delete_proposal(proposal as *mut ProposalRust)
 }
+
+/// delete the pointer
+///
+/// # Safety
+///
+/// This function dereference raw pointers. Even though
+/// the function checks if the pointers are null. Mind not to put random values
+/// in or you may see unexpected behaviors
+#[no_mangle]
+pub unsafe extern "C" fn iohk_jormungandr_delete_fragment(fragment: FragmentPtr) {
+    fragment_delete(fragment as *mut FragmentRust);
```

Comment:
Add error handling?

Reply:
that function can't fail actually, I left the `Result` type by accident, needs to me removed actually.